src/main/java/com/spring/amazon/service/impl/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package com.spring.amazon.service.impl;
+
+import com.spring.amazon.service.FileReaderService;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.stereotype.Service;
+
+@Service
+public class FileReaderServiceImpl implements FileReaderService {
+    @Override
+    public List<String> read(String path) {
+        List<String> result = new ArrayList<>();
+        try (BufferedReader br = new BufferedReader(new FileReader(path))) {
```

Comment:
br - is not informative naming

Reply:
It is only in try block. Anyway, I changed

Reply:
I know, and it looks ok for now, but your methods have tendency to grow, so it's better to write correct naming from the beginning, not to refactore code in future